nservicebus/service-fabric/transaction-sharing.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Service Fabric Persistence Transaction Sharing
+reviewed: 2017-03-20
+component: ServiceFabricPersistence
+---
+
+Service Fabric Persistence exposes the current storage transaction via the `SynchronizedStorageSession` property on the `IMessageHandlerContext` implementation. The transaction can be used to ensure atomicity of operations performed by both the business logic and the persister. This can be especially helpful when running endpoints with the [Outbox](/nservicebus/outbox/) feature turned on.
```

Comment:
> This can be especially helpful when running endpoints with the [Outbox](/nservicebus/outbox/) feature turned on.

to 

> This enables running endpoints with the [Outbox](/nservicebus/outbox/) feature turned on.